.github/workflows/release.yaml

```diff
@@ -3,11 +3,12 @@ name: vSphere Charts
 on:
   push:
     branches:
-      - main
+      - release
 
 jobs:
   release:
     runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/heads/release')
```

Comment:
See this comment: https://github.com/rancher/vsphere-charts/pull/5#issuecomment-1040721772
This should be good to merge once you verify that this will only get triggered for PRs made to the release branch.

Reply:
Tested it on  https://github.com/rayandas/vsphere-charts-1/releases.
It's working fine. Merging this.